### ISSUE TITLE
har-validator version to 5.1.2 not available

### ISSUE THREAD

**Issue by rubensa** [jhipster/generator-jhipster#8843]
##### **Overview of the issue**

**JHipster Generator 5.6.1** depends on **har-validator 5.1.2** which is not available on npm registry any more:

```bash
error 404 Not Found: har-validator@https://registry.npmjs.org/har-validator/-/har-validator-5.1.2.tgz
```

##### **Motivation for or Use Case**

Use JHipster Generator 5.6.1

##### **Reproduce the error**

To reproduce the error you must delete your npm cache (~/.npm on Posix or~/npm-cache on Windows) before doing a ```npm install generator-jhipster```

##### **Related issues**

There are other projects affected by this har-validator version removal from npm registry:
https://github.com/request/request/pull/3055

There is is an issue oppened on har-validator project (looks like someone ran ```npm unpublish har-validator@5.1.2``` after the package was originally published):
https://github.com/ahmadnassri/node-har-validator/issues/113

Unpublish confirmation:
https://github.com/ahmadnassri/node-har-validator/issues/112#issuecomment-437378269

##### **Suggest a Fix**

Upgrade dependency version to 5.1.3

**UPDATE**: As this is the dependency chain for har-validator:
```bash
└─┬ generator-jhipster@5.6.1
  └─┬ insight@0.10.1
    └─┬ request@2.88.0
      └── har-validator@5.1.2 
```
Looks that is insight dependency version which must be updated (if an updated version exists, which I doubt as there is no new har-validator version).

##### **JHipster Version(s)**

5.6.1

##### **Browsers and Operating System**

Linux 4.15.0-39-generic Ubuntu SMP Tue Oct 23 15:48:01 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

-   [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by rubensa** [comment]
@jdubois thanks for checking.

I keep receiving a **404 Not found** for the https://registry.npmjs.org/har-validator/-/har-validator-5.1.2.tgz (do you have a cache proxy? ).

But just realized that problem only affect projects with existing **package-lock.json** (as version is fixed by it).  If I remove **package-lock.json** and install **generator-jhipster** then the dependency is:
```bash
  └─┬ insight@0.10.1
    └─┬ request@2.88.0
      └── har-validator@5.1.3 
```

**Comment by jdubois** [comment]
Yes they've just released version 5.1.3 so that should fix the problem - so there's nothing to do on our side.
Then I don't have a cache at all, so I'm surprised I can download 5.1.2 and not you - I'm doing this directly with my browser, so there can't be any misconfiguration here.

I'm closing this as this should be OK, then I expect quite a lot of people to come over this issue. NPM is quite a mess sometimes!
